cogs/commands.py

```diff
@@ -376,6 +376,30 @@ async def suggestion(self, ctx, *, suggestion: str):
         msg = await ctx.send(embed=em)
         await msg.add_reaction('👍')
         await msg.add_reaction('👎')
+                       
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, payload):
+        user = payload.member
+        if user.bot:return
+        msg = await self.bot.get_guild(payload.guild_id).get_channel(payload.channel_id).fetch_message(payload.message_id)
+        emoji = payload.emoji
+        users = []
+        if msg.author.bot and ("👍"and"👎")in [str(i)for i in msg.reactions]:
```

Comment:
You are using the 👍 and 👎 quite a few times, so i would probably make them a constant, so that they can be changed to plus_one and minus_one or some other emoji easily in future :D